std_msgs/Header header
float64[] left_glove_state
float64[] right_glove_state
